packages/sdk/test/unit/apiClient/certificateAuthProvider.spec.ts

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { assert, expect, use as chaiUse } from "chai";
+import {
+  CertificateAuthProvider,
+  createPemCertOption,
+  createPfxCertOption,
+  ErrorCode,
+  ErrorWithCode,
+} from "../../../src";
+import * as chaiPromises from "chai-as-promised";
+
+chaiUse(chaiPromises);
+
+describe("CertificateAuthProvider Tests - Node", () => {
+  it("should add PEM cert to https agent", async function () {
+    const bearerTokenAuthProvider = new CertificateAuthProvider({
```

Comment:
bearerTokenAuthProvider -> certificateAuthProvider?

Reply:
thank you, updated